virtool/tasks/pg.py

```diff
@@ -42,17 +41,16 @@ async def get(pg: AsyncEngine, task_id: int) -> dict:
     return document
 
 
-async def register(pg, task_client: TasksClient, task_class, context: dict = None, interval: int = None) -> dict:
+async def register(pg, task_class, context: dict = None) -> dict:
     """
     Create a new task record and insert it into SQL databse.
 
     Add the new task to TaskRunner.
```

Comment:
> Add the new task to TaskRunner

No longer accurate.

Reply:
Changes already made in commit [2037eae](https://github.com/virtool/virtool/pull/2182/commits/2037eaefe5b49b20c2d0fdec71d1d30af7dda3ac)